src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] colors = new String[]

```

Comment:
```suggestion
    private static final String[] COLORS = new String[]

```

в именах констант все буквы пишуться большими: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names